gulpfile.js

```diff
@@ -477,9 +479,9 @@ gulp.task('build:client', ['browserify:app', 'concatenate:vendor_js', 'concatena
 gulp.task('build', done => runSequence(
   'production',
   // 'clean:admin',
-  'clean:overrides',
   'clone:default_plugins',
   'build:client',
+  'clean:overrides',
```

Comment:
@volterra79 related to: https://github.com/g3w-suite/g3w-client/pull/325/commits/05659cc4667e46c82b7d531573d3be0411c742bb

To me either way (ie. before or after that change), it always cleans all the contents of the overrides folder.

Maybe when you tried you had a lock on the `static/client/js/app.min.js` file because it was opened somewhere else locally?